python/py-ptpython/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               select 1.0
+
+name                    py-ptpython
+version                 3.0.23
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 BSD
+maintainers             {@catap korins.ky:kirill} openmaintainer
+
+description             Python REPL build on top of prompt_toolkit
+long_description        {*}${description}
+
+homepage                https://github.com/prompt-toolkit/ptpython
+
+checksums               rmd160  88d712e3864658351a0d89b559d8de6aa1681908 \
+                        sha256  9fc9bec2cc51bc4000c1224d8c56241ce8a406b3d49ec8dc266f78cd3cd04ba4 \
+                        size    68146
+
+python.versions         38 39 310 311
+
+python.pep517           yes
+
+python.link_binaries    no
```

Comment:
this is uncommon; why would you want this switched to no? I would expect this to be "yes" and then there would ideally no need for the "ln" stuff in the  "post-destroot"?

Reply:
@reneeotten to compatibility with layout that is created by current `ptpython` port.

Reply:
IMO this should just behave as any other py-XY port. The binaries should go into ${prefix}/bin with the version suffixed. To select whatever version the user wants, one can use the "select" port. 

But perhaps I am not understanding it correctly...?

Reply:
@reneeotten right now it creates a layout:
```
/opt/local/bin/ptipython
/opt/local/bin/ptipython3
/opt/local/bin/ptipython3.10
/opt/local/bin/ptpython
/opt/local/bin/ptpython3
/opt/local/bin/ptpython3.10
```

So, I've moved `ptpython` and `ptpython3` to select's group and construct `ptpython3.10` by hand.

Reply:
@reneeotten anyway, if you think that it is not nessasary I gladly remove that hack.

Reply:
To make things simpler => I've removed that ugly hack. If anyone complains, will do something, but not yet.